chsdi/models/vector/uvek.py

```diff
@@ -255,26 +255,32 @@ class ZAEHLSTELLENUEBER(Base, Vector):
 
 
 class unf_pers_alle(Base, Vector):
-    __tablename__ = 'unf_pers_full'
+    __tablename__ = 'unf_pers_alle'
     __table_args__ = ({'schema': 'astra', 'autoload': False})
     __template__ = 'templates/htmlpopup/astra_unfaelle.mako'
     __bodId__ = 'ch.astra.unfaelle-personenschaeden_alle'
     __label__ = 'accidenttype_de'
+    __timeInstant__ = 'accidentyear'
+    __queryable_attributes__ = ['accidenttype_de', 'accidenttype_fr', 'accidenttype_it', 'accidenttypecode', 'accidentday_de', 'accidentday_fr', 'accidentday_it', 'accidentyear', 'severitycategory_de', 'severitycategory_fr', 'severitycategory_it', 'severitycategorycode', 'roadtype_de', 'roadtype_fr', 'roadtype_it', 'roadtypecode', 'canton', 'fsocommunecode']
```

Comment:
It's a bit ugly to have query_able attributes by language. Backend should abstract this (we should only expose accidenttype and search all language columns). Anyway, that's probably more than we should handle in this PR.

@loicgasser Thoughts?